src/AstroBase.jl

```diff
@@ -1,5 +1,27 @@
 module AstroBase
 
+using Rotations
+export celestial_to_intermediate
 # package code goes here
+"""
+    celestial_to_intermediate(x, y, s)
+
+Converts the celestial to intermediate-frame-of-date matrix given the CIP
+X,Y and the CIO locator s.
+
+```jldoctest
+julia> celestial_to_intermediate(0.2, 0.2, 0.1)
+3×3 RotZYZ{Float64}(0.785398, 0.286757, -0.885398):
+  0.976728   0.0774803  0.2
+ -0.11811    0.972651   0.2
+ -0.179034  -0.218968   0.959166
+```
+"""
+function celestial_to_intermediate(x, y, s)
+    r2 = x^2+y^2
+    e = r2 > 0.0?atan2(y,x):0.0
```

Comment:
My comment was meant to be read "add whitespace around operators". Sorry 🙈

Reply:
oh! doing it!